[Workers] Sunder Workers starter template (#2090)
cloudflare/cloudflare-docs: products/workers/src/content/get-started/quickstarts.md

  repo="cloudflare/worker-template-router"
/>

<WorkerStarter
  title="Sunder Starter (Typescript)"

>   description="A batteries-included starter template using Sunder, Typescript, ESBuild, Jest and Sass. Uses Worker Sites for static assets."

Reviewer: ```suggestion
  description="A batteries-included starter template using Sunder, TypeScript, esbuild, Jest and Sass. Uses Worker Sites for static assets."
```

>   description="A batteries-included starter template using Sunder, TypeScript, esbuild, Jest and Sass. Uses Worker Sites for static assets."

Reviewer 2: ```suggestion
  description="A batteries-included starter template using Sunder, TypeScript, esbuild, Jest, and Sass. Uses Worker Sites for static assets."
```

Reviewer 2: ```suggestion
  description="A complete starter template using Sunder, TypeScript, esbuild, Jest and Sass. Uses Worker Sites for static assets."
```

Reviewer 2: Not sure if mine is the best term, but I don't think `batteries-included` won't generalize well to international audiences.
